WordPress/gutenberg
# Fixed a problem with double encoding of URLs.

## Description
Fixed: #32837

This is a fix for slugs that are URL-encoded strings; it fixes a problem where URLs were being converted twice, destroying the URL.

## How has this been tested?

1. Create Page, with just title. Title can be anything. e.g. ポートフォリオ but slug must also be something that will get URL encoded.
1. Go to either template editor or post editor
1. Insert Navigation block, horizontal or vertical, then choose "start from blank"
1. Insert Page Link block, choose the page created in step 1.
1. Save and check on the frontend.
1. Expected behaviour

## Screenshots 
![スクリーンショット 2021-06-21 12 02 14](https://user-images.githubusercontent.com/1908815/122701629-a32db900-d288-11eb-9d07-d6c7b5a06b27.png)
![スクリーンショット 2021-06-21 12 02 33](https://user-images.githubusercontent.com/1908815/122701636-a4f77c80-d288-11eb-88ca-e39527b5f203.png)


## Types of changes
Bug fix

## Checklist:
